core/common/bytestr.hpp

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_COMMON_BYTESTR_HPP
+#define KAGOME_COMMON_BYTESTR_HPP
+
+#include <libp2p/common/bytestr.hpp>
+#include <libp2p/common/span_size.hpp>
+#include <string_view>
+
+namespace kagome {
+  using libp2p::bytestr;
+
+  inline std::string_view bytestr(const gsl::span<const uint8_t> &s) {
```

Comment:
I find it confusing that it has the same name as the libp2p function but does the opposite thing.

Reply:
It's ok that they have same name, because they do same inconvenient conversion (char and byte pointers).
This second function will be moved to libp2p later.

Reply:
I suggest byte2str.

Reply:
I'd prefere the libp2p one to be renamed to str2byte and this one to be renamed to byte2str, but if you don't want to change libp2p for now, strbyte can be used just for visual distinction.